andela/ah-jumanji-frontend
# #163646664 Show number of follow relationship on profile

#### What does this PR do?
Show the number of followers and following on profile

#### Description of Task to be completed?
Have the number of  followers and following on profile

#### How should this be manually tested?
1. Run the following commands in Terminal:
```
git clone [https://github.com/andela/ah-jumanji-frontend.git](https://github.com/andela/ah-jumanji-frontend.git)
git checkout ft-show-follow-relationship-on-profile-#163646664
touch .env
vi .env
```
2. Edit the`.env` file to add:
```
FIREBASE_API_KEY=AIzaSyDLqlYrfTIkjXb01oA_9svo107jkV-YzAg
FIREBASE_AUTH_DOMAIN=ah-jumanji.firebaseapp.com
SOCIAL_AUTH_API_URL=https://ah-jumanji-staging.herokuapp.com/api/users/social/auth
ARTICLES_API_URL=https://ah-jumanji-staging.herokuapp.com/api/articles/
BASE_URL=https://ah-jumanji-staging.herokuapp.com
```
3. Continue in `Terminal`:
```
npm install
npm start
```
4. Login and view your profile to check on the number of followers.
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#163646664](https://www.pivotaltracker.com/story/show/163646664)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/25703581/52329888-f333ba00-2a04-11e9-82f8-0e3e4f663b74.png)

#### Questions:

#### Checklist:
